.github/workflows/build.yml

```diff
@@ -101,12 +101,8 @@ jobs:
           echo "consul binary type is ${CONSUL_BINARY_TYPE}"
           echo "consul copyright year is ${CONSUL_COPYRIGHT_YEAR}"
           cd ui && make && cd ..
-          mkdir pkg
-          mv ui/packages/consul-ui/dist pkg/web_ui
-
-      - name: Build static-assets
-        run: make static-assets
-
```

Comment:
Previously we were making an intermediate well-known path `pkg/web_ui` which was used by `make static-assets` which dumped the compiled files into `agent/uiserver`.

Now that we are removing `make static-assets` we can move files from the ui directory straight into `agent/uiserver`